commec/utils/dict_utils.py

```diff
@@ -3,17 +3,25 @@
 """
 Static functions useful for dealing with common dictionary tasks.
 """
+import logging
+logger = logging.getLogger(__name__)
 
 @staticmethod
 def deep_update(to_update: dict, has_updates: dict):
```

Comment:
Can we add type annotations here (just because it is a rather odd return signature)

```suggestion
def deep_update(to_update: dict[str, Any], has_updates: dict[str, Any]) -> tuple[dict, list[str]]:
```